build.sbt

```diff
@@ -55,8 +55,7 @@ lazy val root = Project("spark-redshift", file("."))
       // A Redshift-compatible JDBC driver must be present on the classpath for spark-redshift to work.
       // For testing, we use an Amazon driver, which is available from
       // http://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html
-      "com.amazon.redshift" % "jdbc41" % "1.2.27.1051" % "test" from "https://s3.amazonaws.com/redshift-downloads/drivers/jdbc/1.2.27.1051/RedshiftJDBC41-no-awssdk-1.2.27.1051.jar",
-
+      "com.amazon.redshift" % "jdbc42" % "2.1.0.14" % "test" from "https://s3.amazonaws.com/redshift-downloads/drivers/jdbc/2.1.0.14/redshift-jdbc42-2.1.0.14.jar",
```

Comment:
This should get ignored anyways bc we pull it in flame, but yeah good to keep in sync

when you use flame shell, what do you have in this dir?

~/flame/assembly/target/scala_2.12/jars

it should be the redshift-jdbc 2.x jar, not 1

Reply:
also when testing with shell, make sure to clean in between

Reply:
Yeah it is 2.x jar, not 1.
The dependency issue is in spark-redshift, so I think we still need to upgrade it here when building? Or we override dependency for other jar (spark-redshift) when building spark?

Reply:
https://github.com/ActionIQ/flame/blob/bd1b79515086e32737cb6b20bade5fd17d7dc6ce/assembly/pom.xml#L305-L309 is what decides the version that gets installed on the cluster, which is what should be used